internal/rbd/rbd_attach.go

```diff
@@ -207,21 +225,36 @@ func setRbdNbdToolFeatures() {
 		_, stderr, err = util.ExecCommand(context.TODO(), "modprobe", moduleNbd)
 		if err != nil {
 			hasNBD = false
-			log.WarningLogMsg("rbd-nbd: nbd modprobe failed (%v): %q", err, stderr)
+			log.WarningLogMsg("nbd modprobe failed (%v): %q", err, stderr)
 		}
 	}
-
-	stdout, stderr, err := util.ExecCommand(context.TODO(), rbdTonbd, "--help")
-	if err != nil || stderr != "" {
-		hasNBD = false
-		log.WarningLogMsg("running rbd-nbd --help failed with error:%v, stderr:%s", err, stderr)
+	if hasNBD {
+		log.DefaultLog("nbd module loaded")
 	}
 
-	if strings.Contains(stdout, "--cookie") {
-		hasNBDCookieSupport = true
+	// fetch the current running kernel info
+	release, err := util.GetKernelVersion()
+	if err != nil {
+		log.WarningLogMsg("fetching current kernel version failed (%v)", err)
```

Comment:
return from here as we dont need to check cookie support as getting kernel version failed.